internal/cmd/wait.go

```diff
@@ -134,7 +134,7 @@ whether all pods that match the configured prefix to become ready.`,
 				}
 
 				if len(list) == 0 {
-					return true, nil
+					return false, nil
```

Comment:
if the `checkFunc` will always return false, do you need to return `boolean`, shouldn´t this be returning only an `error` ?

Reply:
Thanks for spotting this. I will have a look.

Reply:
The condition did not take into account that there should be pods that match the condition and all of them have to report ready status.